lib/Sema/CSSimplify.cpp

```diff
@@ -6802,11 +6804,16 @@ performMemberLookup(ConstraintKind constraintKind, DeclNameRef memberName,
       }
 
       if (objectType->mayHaveMembers()) {
+        // If there are viable members directly on `Optional`, let's
+        // prioritize them any mark any results found on wrapped type
```

Comment:
typo? :) 
```suggestion
        // prioritize them and mark any results found on wrapped type
```

Reply:
Ah yes, I'll let compat suite run and fix it afterwards.

Reply:
Done!